weaviate/helm/weaviate/values.yaml.tpl

```diff
@@ -12,6 +20,20 @@ ingress:
       - {{ .Values.hostname }}
 
 weaviate:
+  authentication:
+    anonymous_access:
+      enabled: false
+    apikey:
+      enabled: true
+      allowed_keys:
+        - {{ $key }}
+      users:
+        - {{ .Values.adminEmail }}
+  authorization:
+    enabled: true
+    adminlist:
+      users:
+        - {{ .Values.adminEmail }}
   backups:
```

Comment:
i'm pretty sure this will have issues outside gcp since it'll set that entire map to null if you template it as

```
backups:
```

Reply:
Good catch, moved condition